mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -0,0 +1,77 @@
+import {useState} from "react";
+import {renderSpan} from "src/component/editableText/renderSpan";
+import {Input} from "src/component/input/Input";
+
+/**
+ * Cell item props
+ */
+interface EditableTextProps {
+
+  /**
+   * Cell item's text
+   */
+  text: string;
+
+  /**
+   * Function that update element on Enter click or unfocused
+   */
+  onChangeFinish: (value: string) => void;
+}
+
+/**
+ * Render Input or span depend on client actions
+ */
+export const EditableText = (props: EditableTextProps) => {
```

Comment:
Looks like editable text and editable number has no much different logic. Should it be the only one component with prop type: 'text' |'number'?

Reply:
There are using different type for input

Reply:
Done